go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -168,10 +172,17 @@ func (c *Controller) Run(workers int, stopCh <-chan struct{}, runRepair, useLBGr
 		return err
 	}
 	// We need node cache to be synced first, as we rely on it to properly reprogram initial per node load balancers
+	// No need to sync services here, since workers aren't up yet.
```

Comment:
qq: we were not syncing services here anyways right? you have just added this comment to explicitly call that out, that's all right?

Reply:
we were doing it before, but now that I removed it, the comment alone doesn't tell you the whole story. Let me remove it from here and add it to `RequestFullSync`, where it makes more sense.